config.js

```diff
@@ -4,7 +4,7 @@
 module.exports = {
```

Comment:
Just spotted this. I'm curious to know why you would do `const config = {...}; module.exports = config;` over exporting the object directly.

What would you see as the benefit?